application/controllers/HostController.php

```diff
@@ -2,23 +2,60 @@
 
 namespace Icinga\Module\Businessprocess\Controllers;
 
+use Icinga\Application\Modules\Module;
+use Icinga\Module\Businessprocess\IcingaDbObject;
+use Icinga\Module\Businessprocess\ProvidedHook\Icingadb\IcingadbSupport;
+use Icinga\Module\Icingadb\Model\Host;
 use Icinga\Module\Monitoring\Controller;
 use Icinga\Web\Url;
+use ipl\Stdlib\Filter;
 
 class HostController extends Controller
 {
+    protected $isIcingadb;
+
+    protected $explicitIcingadb;
```

Comment:
You shouldn't name this **explicit**Icingadb. It's the exact opposite: *implicit*

Anyway, what about the actual meaning: `$isIcingadbPreferred`

And add documentation!